src/tools-evaluation.md

```diff
@@ -14,6 +14,22 @@ of using Linux to boot into an operating system.
 
 ## Root filesystem
 
+Linux needs a root filesystem with at least one binary that is called
+[`init`](https://docs.kernel.org/admin-guide/init.html). Since booting a system
```

Comment:
This links to a page documenting "high-level reasons for failure to load the init binary", but does not explain what `init` is. Was that intentional?

Reply:
Yea that was intentional; that docs page isn't perfect as it is right now, but at least the best first-hand reference I could find.

Reply:
I find this one particularly helpful because the initramfs is crucial and a common issue for crashes during bringup development. I've run into this a handful of times myself. A bit more elaboration on the potential issue would help here, I think.